src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,476 @@
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    FACET name 
+    SINCE 1 day ago
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago faceted by name"
+  alt="A screenshot of a query that selects the average duration since 1 day ago faceted by name"
+  src={queriesnrqltutorial17}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    By default, a faceted query will return the top 10 results, but you can customize how many results are returned with by placing a  `LIMIT`. In this example, we will retrieve the top 5 results displayed on a line chart with `TIMESERIES`.
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    FACET name 
+    SINCE 3 hours ago 
+    LIMIT 5 
+    TIMESERIES
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration from the top 5 results since 3 hours ago grouped by name with a timeseries"
+  alt="A screenshot of a query selecting the average duration from the top 4 results since 3 hours ago grouped by name with a timeseries"
+  src={queriesnrqltutorial18}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    But maybe you don't want a line chart or want a larger list of transactions. By removing the `TIMESERIES`, we can instead render a bar or pie chart, and by increasing the `LIMIT`, we can increase the scope of the list.
+    ```sql
+    SELECT average(duration) 
+    FROM Transaction 
+    FACET name 
+    SINCE 3 hours ago 
+    LIMIT 20
+    ```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration from the top 20 results since 3 hours ago grouped by name without a timeseries"
+  alt="A screenshot of a query selecting the average duration from the top 20 results since 3 hours ago grouped by name without a timeseries"
+  src={queriesnrqltutorial19}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+  Finally, here's a slightly more complex query that compares the quantity of Web transactions, broken down by individual applications that report to New Relic:
+  ```sql
+  SELECT count(*) 
+  FROM Transaction 
+  WHERE transactionType='Web' 
+  FACET appName 
+  LIMIT 5 
+  SINCE 6 hours ago 
+  TIMESERIES
+  ```
+  </Side>
+  <Side>
+    <img
+  title="Select all web transactions from the top 5 results since 6 hours ago grouped by appName with a timeseries"
+  alt="A screenshot of a query selecting all web transactions from the top 5 results since 6 hours ago grouped by appName with a timeseries"
+  src={queriesnrqltutorial20}
+/>
+  </Side>
+</SideBySide>
+Grouping or "faceting" is important: it allows us to get dimensional with our data. Using the `FACET` keyword is the simplest way to do this. When applying an aggregation function, we can use `FACET` to group by any attribute, default or custom, that reports event data (just like the `WHERE` clause). We recommend tagging on custom data to your events. This allows you to more easily investigate the resulting data set.
```

Comment:
the 'tagging on' is a bit ambiguous and potentially confusing, as it overlaps with 'tags'. I'd say maybe just 'We recommend adding custom attributes to your data, or reporting new custom events...' and maybe linking to those docs.

Reply:
Good idea, thank you!